astroquery/alma/core.py

```diff
@@ -90,16 +100,12 @@ def query_region_async(self, coordinate, radius, cache=True, public=True,
         kwargs : dict
             Passed to `query_async`
         """
-        coordinate = commons.parse_coordinates(coordinate)
-        cstr = coordinate.fk5.to_string(style='hmsdms', sep=':')
-        rdc = "{cstr}, {rad}".format(cstr=cstr, rad=coord.Angle(radius).deg)
-
-        if payload is None:
+        coordinates = commons.parse_coordinates(coordinate)
+        if not payload:
```

Comment:
this doesn't seem like a necessary change; `payload is None` is more specific than `not payload`